packages/yourloops/pages/notifications/notifications-list.tsx

```diff
@@ -106,6 +107,7 @@ const NotificationsPage: FunctionComponent<PropsWithChildren> = () => {
                     notification={notification}
                     userRole={user.role}
                     onHelp={handleSwitchRoleOpen}
+                    refreshNotificationHook={refreshNotificationHook}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
                    refreshReceivedInvitations={refreshReceivedInvitations}
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏So passing this function as a prop does not create a memory leak. But directly calling `await notificationsHook.refreshReceivedInvitations()` from the child component would? Am I correct?

Reply:
yes it is

Reply:
okay :+1: